modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java

```diff
@@ -166,6 +150,58 @@ private MockLiferayResourceRequest _createMockLiferayResourceRequest(
 		return mockMultipartHttpServletRequest;
 	}
 
+	private void _exportImportObjectDefinition(
+			String fileName, String name, boolean system)
+		throws Exception {
+
+		PermissionChecker permissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		PermissionThreadLocal.setPermissionChecker(
+			PermissionCheckerFactoryUtil.create(TestPropsValues.getUser()));
```

Comment:
@pedro-oliveira446 we have another pattern to handle the permissionChecker, [see](https://github.com/liferay/liferay-portal/blob/f6d10ea9c662b8d5408636c2805681d249dd0a1a/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionServiceTest.java#L78-L77),

could you try to apply? it'll simplify the test and keep the pattern around the portal.

Thanks!